context/app/static/js/components/entity-search/sidebar/ConfigureSearchTable/ConfigureSearchTable.jsx

```diff
@@ -19,47 +20,56 @@ function ConfigureSearchTable({
   availableFields,
 }) {
   const { numericFacetsProps } = useStore();
+  const [searchBarFieldName, setSearchBarFieldName] = useState('');
+
+  const handleChange = (event) => {
+    setSearchBarFieldName(event.target.value);
+  };
+
   return (
-    <TableContainer component={Paper}>
-      <Table>
-        <TableHead>
-          <TableRow>
-            <TableCell />
-            <TableCell>Add Facet</TableCell>
-            <TableCell>Add Column</TableCell>
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          {excludeDateFieldConfigs(getFieldEntriesSortedByConfigureGroup(availableFields)).map(
-            ([fieldName, fieldConfig]) => (
-              <TableRow key={fieldName}>
-                <TableCell>{fieldConfig.label}</TableCell>
-                {['string', 'boolean'].includes(fieldConfig.type) || numericFacetsProps?.[fieldName] ? (
+    <FlexGrow>
+      <StyledSearchBar fullWidth value={searchBarFieldName} onChange={handleChange} />
+      <TableContainer component={Paper}>
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableCell />
+              <TableCell>Add Facet</TableCell>
+              <TableCell>Add Column</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {filterFieldConfigs(getFieldEntriesSortedByConfigureGroup(availableFields), searchBarFieldName).map(
+              ([fieldName, fieldConfig]) => (
+                <TableRow key={fieldName}>
+                  <TableCell>{fieldConfig.label}</TableCell>
+                  {['string', 'boolean'].includes(fieldConfig.type) || numericFacetsProps?.[fieldName] ? (
+                    <TableCell>
+                      <Checkbox
+                        checked={fieldName in selectedFacets}
+                        size="small"
+                        color="primary"
+                        onChange={(event) => handleToggleFacet(event, fieldConfig)}
+                      />
```

Comment:
Could you drop the ternary, since both sides begin with `<TableCell>`, and just put a the checks in front of `<Checkbox>`? No worries: I could be confused.